src/capi.cpp

```diff
@@ -1098,6 +1098,74 @@ int rtcAddTrackEx(int pc, const rtcTrackInit *init) {
 	});
 }
 
+RTC_C_EXPORT int rtcAddAACTrack(int pc, const rtcTrackInit *init, uint8_t *latmMuxCfg, int size) {
+	return wrap([&] {
+		auto peerConnection = getPeerConnection(pc);
+
+		if (!init)
+			throw std::invalid_argument("Unexpected null pointer for track init");
+
+		auto direction = static_cast<Description::Direction>(init->direction);
+
+		string mid;
+		if (init->mid) {
+			mid = string(init->mid);
+		} else {
+			switch (init->codec) {
+			case RTC_CODEC_AAC:
+				mid = "audio";
+				break;
+			default:
+				mid = "audio";
+				break;
+			}
+		}
+
+		optional<Description::Media> optDescription = nullopt;
+		switch (init->codec) {
+		case RTC_CODEC_AAC: {
+			auto desc = Description::Audio(mid, direction);
+			switch (init->codec) {
+			case RTC_CODEC_AAC:
+					if (latmMuxCfg) {
+						std::string muxbuf="cpresent=0;config=";
+						for (int i = 0; i < size; i++) {
+							char buf[3] = {0};
+							snprintf(buf, 3, "%02x", latmMuxCfg[i]);
+							muxbuf.append(buf, 2);
+						}
+						
+						desc.addAudioCodec(init->payloadType, "MP4A-LATM", muxbuf);
```

Comment:
Could you please create a dedicated method `Description::Media::addAacCodec()`? The logic to serialize `latmMuxCfg` could also be moved there.

Reply:
done, add  Description::Media::addAacCodec method